From 8884914546ee68a440342587c4a7c9189dbc8443 Mon Sep 17 00:00:00 2001
From: Steve Lhomme <robux4@videolan.org>
Date: Wed, 12 Sep 2018 11:18:15 +0200
Subject: [PATCH] [film-grain] copy user_priv from the source img

Since we don't return the original img we need to prive the same user_priv
data as the aom_image_t that would be returned without film-grain.

Change-Id: I6ac6c575b9ca34673018cf7533955afe4996e6fc
---
 av1/av1_dx_iface.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/av1/av1_dx_iface.c b/av1/av1_dx_iface.c
index 8899556ca..f5e19ea5c 100644
--- a/av1/av1_dx_iface.c
+++ b/av1/av1_dx_iface.c
@@ -642,6 +642,7 @@ static aom_image_t *add_grain_if_needed(aom_image_t *img,
       *grain_img_ptr = NULL;
     }
   }
+  grain_img_buf->user_priv = img->user_priv;
 
   return grain_img_buf;
 }
-- 
2.17.0.windows.1

